recipes/backports.cached_property/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "backports.cached-property" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a7a390f4c006db13a3eb268f79dc0fd435bd0739bb570468a90979ef7b1bfce2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python  <=3.7
```

Comment:
Is there any reason this is incompatible with installing on Python >= 3.8? Being able to install it regardless would help downstream noarch packages.

Reply:
Please correct me if I am wrong, but this being a backports package, it should not be used with Python >= 3.8. And downstream packages might have a conditional dependency like:

```
requirements:
  run:
    - backports.cached-property  // Python <= 3.7
```

Reply:
It doesn't conflict with the standard library name on Python >= 3.8 and upstream [advertises support](https://github.com/penguinolog/backports.cached_property/blob/c633c113e4123ab3dfd4e0766c97d97405d849ea/setup.py#L33-L37) for >= Python 3.8. In the absence of some conflict, I don't see any rationale why the conda-forge recipe should be more restrictive than upstream.

Regarding downstream, my point is that packages that rely on this, if they want to be no arch, are not allowed to have any conditional dependencies. Leaving this package installable on >= 3.8 allows those packages to avoid needing the conditional dependency, like we already do for e.g. importlib-metadata.

Reply:
> if they want to be no arch, are not allowed to have any conditional dependencies

Oh I didn't know that, what is the reason for this constraint?

Well IMHO it seemed cleaner to use Python <=3.7 but I don't want to put the constraint on downstream to not be noarch. I'll update my PR. Thanks!